app/helpers/loans_helper.rb

```diff
@@ -2,4 +2,29 @@ module LoansHelper
   def back_to_loans
     session[:loans_path] || loans_path
   end
+
+  def health_status_message(loan)
+    if loan.loan_health_check.healthy?
+      status_message = {
+        status: "healthy",
+        message: I18n.t("health_status.healthy"),
+        warnings: []
+      }
+    else
+      warnings = loan.loan_health_check.health_warnings.collect {|w| I18n.t("health_status.warnings.#{w}")}
```

Comment:
Please use `map` instead of `collect`. They are the same but `map` is shorter. 

Also please put spaces inside `{}`s.